package.json

```diff
@@ -10,59 +10,47 @@
 	},
 	"homepage": "https://github.com/utily",
 	"private": false,
-	"main": "dist/index.js",
-	"typings": "dist/index.d.ts",
 	"type": "module",
+	"main": "./dist/cjs/index.js",
+	"module": "./dist/mjs/index.js",
+	"types": "./dist/types/index.d.ts",
+	"exports": {
+		".": {
+			"import": {
+				"default": "./dist/mjs/index.js",
+				"types": "./dist/types/index.d.ts"
+			},
+			"require": {
+				"default": "./dist/cjs/index.js",
+				"types": "./dist/types/index.d.ts"
+			}
+		}
+	},
 	"git": {
 		"tagName": "v${version}"
 	},
 	"github": {
 		"release": true
 	},
-	"jest": {
-		"transform": {
-			"^.+\\.(j|t)sx?$": [
-				"ts-jest",
-				{
-					"tsconfig": "./tsconfig.test.json"
-				}
-			]
-		},
-		"transformIgnorePatterns": [
-			"<rootDir>/node_modules/(?!(cryptly|authly|isoly|gracely|cloudly-http|cloudly-router|cloudly-formdata)/.*)"
-		],
-		"testEnvironment": "node",
-		"testRegex": "((\\.|/)(test|spec))(\\.|\\/.+)(jsx?|tsx?)$",
-		"testPathIgnorePatterns": [
-			"node_modules/",
-			"dist/"
-		],
-		"collectCoverageFrom": [
-			"**/*.{ts,tsx,js,jsx}",
-			"!**/node_modules/**",
-			"!**/dist/**"
-		]
-	},
 	"scripts": {
 		"lint": "eslint --max-warnings=0 '**/*.{ts,tsx}'",
 		"fix": "eslint '**/*.{ts,tsx}' --fix",
-		"build": "tsc -p .",
-		"test": "jest",
+		"build": "rimraf dist/* && tsc -p tsconfig.mjs.json && tsc -p tsconfig.cjs.json",
+		"test": "vitest --run",
+		"coverage": "vitest --coverage",
 		"prepare": "npm run build",
 		"clean": "rimraf dist node_modules coverage"
```

Comment:
Add action for checking everything ahead of pushing:
 `"verify": npm run build && npm run test && npm run lint && npm audit`

Reply:
I replaced the build step in the current publish action with calling verify instead.